t/t0095-bloom.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/sh
+
+test_description='test bloom.c'
+. ./test-lib.sh
+
+test_expect_success 'get bloom filters for commit with no changes' '
+	git init &&
+	git commit --allow-empty -m "c0" &&
+	cat >expect <<-\EOF &&
+	Filter:
+	Length:0
+	Data:
+	EOF
+	test-tool bloom get_filter_for_commit "$(git rev-parse HEAD)" >actual &&
+	test_cmp expect actual 
+'
+
+test_expect_success 'get bloom filter for commit with 10 changes' '
+	rm actual &&
+	rm expect &&
+	mkdir smallFolder &&
+	cd smallFolder &&
+	for i in {0..10}
+	do
+		echo $i >$i
+	done &&
+	git add . &&
+	git commit -m "commit with 10 changes" &&
```

Comment:
11 changes with the directory.

Reply:
Include a very clear paragraph about how directories are included in the change count. Perhaps have a test that is a few directories deep? Make it something where 10 * files < 64 * L < 10 * (files + directories) to make it really clear that we are checking the count properly.